packages/babel-preset/index.js

```diff
@@ -53,21 +55,27 @@ function getTargets(
   if (env !== 'production') {
     return target === 'node' ? { node: 'current' } : { esmodules: true };
   }
-  // TODO: distinguish between app and libraries for node as well
+
+  // TODO: Distinguish between app and libraries for node as well.
   if (target === 'node') return { node: '10.0' };
+
   if (
     ignoreBrowserslistConfig ||
-    !loadConfig({ path: configPath || path.resolve('.') })
-  )
-    return { browsers: targets.browsers || defaultBrowsers };
+    !loadConfig({ path: path.resolve(configPath) })
+  ) {
+    return targets || defaultBrowsers;
+  }
 
-  // we don't run browserslist ourself b/c that would require doing a bunch
-  // of additional transforms to get the output in a format preset-env would accept
+  // We don't run browserslist ourself b/c that would require doing a bunch
+  // of additional transforms to get the output in a format preset-env would
+  // accept.
   return targets || undefined;
 }
 
-function preset(_, explicitOptions = {}) {
-  const env = _.env() || 'production'; // default to prod
+function preset(api, explicitOptions = {}) {
+  // FIXME: This is meant to default to production, but Babel actually defaults
```

Comment:
ARGH... we're actually defaulting to `development` because that's what `api.env()` does

Reply:
eww